sdk/healthinsights/azure-health-insights-cancerprofiling/src/samples/java/com/azure/health/insights/cancerprofiling/SampleInferCancerProfile.java

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.health.insights.cancerprofiling;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.OffsetDateTime;
+import java.util.Arrays;
+import java.util.LinkedList;
+
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.polling.PollerFlux;
+import com.azure.health.insights.cancerprofiling.models.PatientInfoSex;
+import com.azure.health.insights.cancerprofiling.models.ClinicalCodedElement;
+import com.azure.health.insights.cancerprofiling.models.PatientInfo;
+import com.azure.health.insights.cancerprofiling.models.PatientRecord;
+import com.azure.health.insights.cancerprofiling.models.PatientDocument;
+import com.azure.health.insights.cancerprofiling.models.ClinicalDocumentType;
+import com.azure.health.insights.cancerprofiling.models.DocumentType;
+import com.azure.health.insights.cancerprofiling.models.DocumentContentSourceType;
+import com.azure.health.insights.cancerprofiling.models.DocumentContent;
+import com.azure.health.insights.cancerprofiling.models.OncoPhenotypeResults;
+import com.azure.health.insights.cancerprofiling.models.OncoPhenotypeModelConfiguration;
+import com.azure.health.insights.cancerprofiling.models.OncoPhenotypeData;
+import com.azure.health.insights.cancerprofiling.models.OncoPhenotypeResult;
+import com.azure.health.insights.cancerprofiling.models.ClinicalNoteEvidence;
+
+public class SampleInferCancerProfile {
```

Comment:
Better named as `InferCancerProfileSample`. Same as another manual written sample.

Reply:
we are releasing public preview this week.
if this is not a blocker, we prefer not to change it now.